src/audio5.js

```diff
@@ -723,8 +735,9 @@
      */
     play: function () {
       if(this.audio) {
-        this.audio.play();
+        var playPromise = this.audio.play();
```

Comment:
Only changed in /src/audio5 as it looks like previous change was made to build output rather than src